Source/ProceduralFairings/FairingBase.cs

```diff
@@ -346,6 +357,9 @@ void SetUIChangedCallBacks()
 
             Fields[nameof(showInterstageNodes)].uiControlEditor.onFieldChanged += OnNodeVisibilityChanged;
             Fields[nameof(showInterstageNodes)].uiControlEditor.onSymmetryFieldChanged += OnNodeVisibilityChanged;
+            
+            //Fields[nameof(openFairing)].uiControlEditor.onFieldChanged += OnToggleOpen;
+            //Fields[nameof(openFairing)].uiControlEditor.onSymmetryFieldChanged += OnToggleOpen;
```

Comment:
Since you're relying on OnEditorShipModified to fire to start redrawing things, and not using these, just remove them.